src/diffpy/utils/scattering_objects/diffraction_objects.py

```diff
@@ -18,6 +19,20 @@
 
 
 class Diffraction_object:
+    """FIXME: Add class docstring.
+
+    .. deprecated:: 3.5.1
```

Comment:
Following the numpy deprecation standards:

<img width="775" alt="Screenshot 2024-11-09 at 12 28 11 PM" src="https://github.com/user-attachments/assets/7c2e4f04-d8e3-4d0d-8afc-fbb0cc6f23c6">

Ref: https://numpydoc.readthedocs.io/en/latest/format.html#deprecation-warning